openerp/addons/base/migrations/7.0.1.3/pre-migration.py

```diff
@@ -128,6 +128,13 @@ def update_base_sql(cr):
             "ALTER TABLE ir_act_client INHERIT ir_actions")
 
 
+def is_module_installed(cr, module):
+    cr.execute(
+        "SELECT id FROM ir_module_module "
+        "WHERE name=%s and state='installed'", (module,))
```

Comment:
I think I'm totally neophyte with OpenUpgrade, so... I can not disagree with you ;) You suggest me to make a PR on OCA/openupgradelib to add this method? Why not.